core/src/main/java/org/apache/camel/kafkaconnector/CamelSinkTask.java

```diff
@@ -122,7 +122,14 @@ public void put(Collection<SinkRecord> sinkRecords) {
             }
             exchange.getMessage().setHeaders(headers);
             exchange.getMessage().setBody(record.value());
-            LOG.debug("Sending {} to {}", exchange, LOCAL_URL);
+
+            if (LOG.isDebugEnabled()) {
```

Comment:
I seem to remember that in modern logging frameworks this is done already by using the logging placeholders syntax (i.e.`LOG.debug("... {}", yourValue);`) so it should be useless and just pollute the code, but I might be wrong.

Reply:
Yep, you are right. It only makes sense if there is string concatenation (to avoid multiple String objects being created when the level is disabled), which is not the case here. Will revert it.